### ISSUE TITLE
[$500] IOU - Unread marker does not appear above IOU preview when IOU is received

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#31621]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.1-7
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:** @

## Action Performed:
1. Navigate to staging.new.expensify.com.
2. [User A] Request money from User B.
3. [User B] Open chat with User A.
Note that there is no unread marker above IOU preview.

## Expected Result:
The unread green line marker appears above IOU preview.

## Actual Result:
The unread green line marker does not appear above IOU preview, while it is marked as unread in LHN.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/9e127227-92b8-407f-a419-3f4da2da6078

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0191eae87d3b1a020d</li>
        <li>Upwork Job ID: 1726942624730722304</li>
        <li>Last Price Increase: 2023-12-18</li>
    </ul>
</details>

**Comment by izarutskaya** [comment]
Not reproducible on production

https://github.com/Expensify/App/assets/115492554/ad106572-3507-4b04-96e5-20029661d74f

**Comment by MonilBhavsar** [comment]
Looks dupe of https://github.com/Expensify/App/issues/29376

**Comment by izarutskaya** [comment]
Hi @danieldoglas,  on prod, the green line also does not appear for new chat. But the green line also does not show for existing chat. Could you check please videos with existing chats, issue is still reproducible for tester.

https://github.com/Expensify/App/assets/115492554/0e94665b-0379-46a8-ad37-f5c5bec280b2

https://github.com/Expensify/App/assets/115492554/30996355-da27-4bce-b1d8-48b44563200f

**Comment by danieldoglas** [comment]
I see, thanks for confirming!

@MonilBhavsar I don't think it's quite a duplicate since it's not happening in production, and that issue you tagged looks a little bit older

**Comment by danieldoglas** [comment]
I agree this is a problem, I don't think we need to block the deploy on it.

**Comment by MonilBhavsar** [comment]
@danieldoglas agree, but we're working on making unread marker consistent for this case. I suggest to put this issue on HOLD for https://github.com/Expensify/App/issues/29376

**Comment by danieldoglas** [comment]
@MonilBhavsar OK, do you want to take care of this one since you're also taking care of #29376 ?

**Comment by MonilBhavsar** [comment]
Happy to do that. I'll assign myself

**Comment by MonilBhavsar** [comment]
Still on HOLD. PR in review

**Comment by stephanieelliott** [comment]
The blocking PR is on staging: https://github.com/Expensify/App/pull/32177

Will take this off hold once it's merged to production!

**Comment by MonilBhavsar** [comment]
Off hold! Applause team cc @izarutskaya could you please help to retest this issue and confirm if it is still reproducible, thanks!

**Comment by MonilBhavsar** [comment]
> Off hold! Applause team cc @izarutskaya could you please help to retest this issue and confirm if it is still reproducible, thanks!

@izarutskaya 👋

**Comment by MonilBhavsar** [comment]
Bumping to weekly as we're waiting for a retest here cc @stephanieelliott

**Comment by stephanieelliott** [comment]
I just went ahead and re-tested so we could unblock the issue. I am unable to reproduce this anymore! Seems like this one is fixed, gonna go ahead and close
